src/proctored-exam-settings/ProctoredExamSettings.test.jsx

```diff
@@ -375,6 +375,47 @@ describe('Hides fields based on user permissions', () => {
   });
 });
 
+describe('Component has focus indicators', () => {
+  beforeEach(async () => {
+    auth.getAuthenticatedHttpClient = jest.fn(() => ({
+      get: async () => ({
+        data: {
+          proctored_exam_settings: {
+            enable_proctored_exams: true,
+            allow_proctoring_opt_out: false,
+            proctoring_provider: 'mockproc',
+            proctoring_escalation_email: 'test@example.com',
+            create_zendesk_tickets: true,
+          },
+          available_proctoring_providers: ['software_secure', 'proctortrack', 'mockproc'],
+          course_start_date: '2070-01-01T00:00:00Z',
+        },
+      }),
+    }));
+
+    auth.getAuthenticatedUser = jest.fn(() => ({ userId: 3, administrator: true }));
+    await act(async () => render(<ProctoredExamSettings {...defaultProps} />));
+  });
+
+  afterEach(() => {
+    cleanup();
+  });
+
+  it('adds focus indicators to checkboxes and radio buttons', async () => {
+    const checkbox = screen.getByLabelText('Enable Proctored Exams');
+    checkbox.focus();
+    // need to get parent of checkbox because that is where the style is being applied
+    const checkboxParent = checkbox.parentElement;
+    expect(window.getComputedStyle(checkboxParent)._values.border).toEqual('1.5px solid #4D90FE');
```

Comment:
The computed style for this element isn't correct, and I'm not exactly sure why. I know that I'm getting the correct element to test, and that the checkbox is still the active element.